Source/ThirdParty/skia/src/xml/SkDOM.cpp

```diff
@@ -179,118 +291,6 @@ const char* SkDOM::AttrIter::next(const char** value) {
 #include "include/private/base/SkTDArray.h"
 #include "src/xml/SkXMLParser.h"
```

Comment:
I'm surprised you didn't have to move the includes too, since SkDOMParser inherits from SkXMLParser that is included here.

Reply:
Clang 19 was very explicit about the definition of `SkDOMParser` being missing when trying to handle the `SkDOM` constructor, so I figured out that shuffling the code around should be enough. The post-commit build bots that use Clang have been happy, e.g. https://build.webkit.org/#/builders/629/builds/15338

Reply:
Yes, I think moving the code is the right solution, but I don't understand why you didn't have to move also the include of SkXMLParser header too that was right before the SkDOMParser class because it inherits from SkXMLParser

Reply:
The `SkDOMParser.h` header is already included at the top of the source file, so this instance can be removed. Actually, the `#include SkTDArray.h` can also be removed because it's included transitively from `SkXMLWriter.h`. I have made an upstream patch with the `#include` statements removed from the middle of the file: https://skia-review.googlesource.com/c/skia/+/930496